uri-schemes.md

```diff
@@ -124,6 +126,34 @@ HTTP Status code: NOT 2XX, idealy the 4XX if it's user error and 5XX if it's the
 
 json object can have other properties too, but currently they are ignored by core.
 
+## **DCLOGIN** <a name="DCLOGIN"></a>
+
+|                          |                       |
+| ------------------------ | --------------------- |
+| Scheme                   | `DCLOGIN:`            |
+| Used for                 | Account setup         |
+| Related Terms\*          | Account Login QR code |
+| Available on             | draft                 |
+| Decoded by the core \*\* | draft                 |
+| Other apps using it      | none, only DeltaChat  |
+
+### Syntax
+
+```
+DCLOGIN:minVersion/CredentialsJSON
+# example:
+DCLOGIN:1/{"email":"me@example.com","password":""}
```

Comment:
should the json schema not also be specified?

Reply:
I forgot to commit it, I just fixed that